### ISSUE TITLE
YAM 2.8 doesn't use signatures and erased mine!

### ISSUE THREAD

**Issue by jens-maus** [jens-maus/yam#370]
**Originally by _hypexed@yahoo.com.au_ on 2013-01-13 15:57:45 +0100**

___
Hello.

I just updated a few days ago from 2.7. But now YAM is annoying me as it erased my signature and once I worked out how to add it back in as the default it still doesn't work. I tried adding another one but that doesn't work either.

I also tried sending myself an email in case it was hidden. Nup, no sign. 

It's getting hard needing to copy my signature from an old sent mail and copy it to a new one everytime I write a mail! Why is it doing this?! :-?

And will I now have to rollback or use a debug version to fix it?

**Comment by jens-maus** [comment]
**Originally on 2013-01-13 18:18:11 +0100**

___
Are you sure you also have configured the identities you have configured in YAM associated to the signature? In 2.8 you have to not only configure the signature in the signature configuration, you also have to associate the signatures with the identity you want to see it used per default.

**Comment by jens-maus** [comment]
**Originally by _hypexed@yahoo.com.au_ on 2013-01-14 06:45:54 +0100**

___
No identities were configured and left for YAM 2.8 to sort out. The result being all but my signature was left intact. How does 2.8 generate the .config file? I think it missed something!

I have two log ins if that makes any difference. But in reality only one is used. I had upgraded all the MUI classes while still using 2.7. I then found 2.8 and so upgraded to that. I rebooted. Loaded up the new YAM. Logged in. All looked fine except when I saw that my signature had been erased.
